qiime_studio/api/jobs.py

```diff
@@ -59,22 +59,11 @@ def create_job():
     action_type = request_body['actionType']
     inputs = request_body['inputs']
     parameters = request_body['parameters']
-    outputs_ = request_body['outputs']
+    outputs = request_body['outputs']
 
     plugin = PLUGIN_MANAGER.plugins[plugin]
     action = getattr(plugin, action_type)[action]
 
-    outputs = collections.OrderedDict()
-    for key, value in action.signature.outputs.items():
-        path = outputs_[key]
```

Comment:
Current diff of the code still need this, as it syncs the order of `outputs` to the signature. Line `136` below assumes this. But, what we can probably do is just use the keys on the result object (because they exist now), so this loop can still go away, but we need to change how it's saved slightly.